uaclient/security.py

```diff
@@ -191,31 +191,21 @@ def __init__(self, client: UASecurityClient, response: "Dict[str, Any]"):
     def id(self):
         return self.response.get("id", "UNKNOWN_CVE_ID").upper()
 
-    def _get_related_notices(self):
-        if hasattr(self, "notices"):
-            return self.notices
-        self.notices = {}
-        for notice_id in self.response.get("notices", []):
-            self.notices[notice_id] = self.client.get_notice(
-                notice_id=notice_id
-            )
-        return self.notices
+    def get_notices_metadata(self):
```

Comment:
@blackboxsw Maybe I am missing something here, but I couldn't figure out what is the new cache mechanism we are introducing here. The old `_get_related_notices` method seems to already covers the cache that we are implementing.

Also, I could not spot that cache mechanism on the USN changes as well

Reply:
> @blackboxsw Maybe I am missing something here, but I couldn't figure out what is the new cache mechanism we are introducing here. The old `_get_related_notices` method seems to already covers the cache that we are implementing.
> 
> Also, I could not spot that cache mechanism on the USN changes as well
sorry about the PR incorrect description (I updated it). This PR only renames `_get_related_notices` to a public method `get_notices_metadata`. 

The `USN.get_cves_metadata` is cached on this instance as `self._cves` instance attr using the same type of mechanism

Reply:
Oh okay, I missed the new method, I was focusing only on the method that were already being used on the code. My mistake